.github/workflows/s2i.yaml

```diff
@@ -0,0 +1,31 @@
+name: Test s2i
+
+on:
+  push:
+    paths:
+      - .github/workflows/s2i.yaml
+      - s2i/**
+  pull_request:
```

Comment:
@garethahealy So one other thing to keep in mind here is that in this repository, the `pull_request` checks will always fail because this action needs access to secrets. Since `pull_requests` only have read access to repositories from forks.. they won't actually have access to the secrets.

There are workarounds for this that I'm going to try to get an example of out for, but for now it might be best to just remove the `pull_request` section from this workflow.